oak_restricted_kernel_interface/src/lib.rs

```diff
@@ -15,8 +15,11 @@
 //
 
 #![no_std]
+#![feature(c_size_t)]
 
 pub mod errno;
+mod raw_syscall;
+pub mod syscall;
 pub mod syscalls;
```

Comment:
There's now three crates related to syscalls, it feels like we should be able to do it with less.
For example, could (a) syscalls (which just contains the numeric identifiers) merged with `raw_syscalls`, or at the very least (b) not made `pub`?

Reply:
I stumbled over this too, but upon consideration I personally think it's fine, having `raw_syscall` on the top levels helps given that it's a deals with code generation. 

`syscalls` list the identifiers
`syscall` contains utilies to make a syscall.